requirements.txt

```diff
@@ -3,4 +3,5 @@ mypy>=0.790
 python_dateutil >= 2.5.3
 setuptools >= 21.0.0
 urllib3 >= 1.25.3
-toml>=0.10.2
\ No newline at end of file
+toml>=0.10.2
+pandas >= 1.2.4
```

Comment:
Shouldn't it be numpy? I don't see pandas usage

Reply:
Both numpy and pandas are heavy dependencies. It is better to avoid them unless they are used to provide functionality that would be hard to get otherwise.

Reply:
The code used to use pandas while I was implementing this, but now it do not use it. Neither numpy nor pandas are required now. This line will be removed.

Reply:
Done